synapse/storage/schema/main/delta/65/05_remove_room_stats_historical_and_user_stats_historical.sql

```diff
@@ -0,0 +1,18 @@
+/* Copyright 2021 The Matrix.org Foundation C.I.C
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ --remove unused tables room_stats_historical and user_stats_historical
```

Comment:
```suggestion
 -- Remove unused tables room_stats_historical and user_stats_historical
 -- which have not been read or written since schema version 61.
```

I don't think this is a problem, but I felt like it would be 'nice to have' a brief summary of why it's being removed.
Otherwise everything LGTM.